core/go/internal/privatetxnmgr/syncpoints/dispatch.go

```diff
@@ -44,12 +41,12 @@ type DispatchPersisted struct {
 	ID                       string             `json:"id"`
 	PrivateTransactionID     string             `json:"privateTransactionID"`
```

Comment:
I couldn't explain to myself as I was working on this:
- Why this is not a `uuid`
- Why there is no foreign key relationship

... but I think @hosie you are working in this area, so I decided not to make any changes.

Reply:
I can't explain why this is not a UUID.  Don't remember what (if) I was thinking at the time of writing the code or maybe I just didn't have a good grasp on what things we were going to be settling on UUIDs for.  Eitherway, agree it should be uuid and should have cascading delete.